components/form/FRadioGroup.vue

```diff
@@ -0,0 +1,173 @@
+<template lang="pug">
+fieldset.FRadioGroup(
+  :class="classes"
+  :disabled="disabled"
+)
+  FField(
+    v-bind="{ name, label, labelTextColor, hint, hideHint, hintTextColor }"
+  )
```

Comment:
regroupable également :
Le `FField`, au lieu de prendre `labelType` en props (qui a été oublié d'ailleurs ici), pourrait de façon plus haut niveau prendre `fieldset: boolean` en prop et i) se rendre comme un component :is fieldset et ii) calculer le label type corespondant

tu en penses quoi ?